src/main/java/org/broadinstitute/hellbender/tools/exome/ExomeReadCounts.java

```diff
@@ -240,6 +249,14 @@
     )
     protected ExonOutInfo exonOutInfo = ExonOutInfo.COORDS;
 
+    @Argument(
+            doc = "Filter duplicate reads",
+            shortName = FILTER_DUPLICATE_READS_SHORT_NAME,
+            fullName = FILTER_DUPLICATE_READS_FULL_NAME,
+            optional = true
+    )
+    protected Boolean filterDuplicateReads = true;
```

Comment:
The one issue is that I would have to negate the if statement where it's used for default behavior.